shared/src/business/useCases/judgeActivityReport/getOpinionsFiledByJudgeInteractor.ts

```diff
@@ -87,5 +87,15 @@ export const getOpinionsFiledByJudgeInteractor = async (
 
   const sortedResult = orderBy(result, 'eventCode', 'asc');
 
-  return sortedResult;
+  await applicationContext.getNotificationGateway().sendNotificationToUser({
```

Comment:
Why did these endpoints need to be transitioned to async?

Reply:
They are being transitioned to async the time restrictions on the lambdas and api gateways is causing the call to crash. Part of the requirements as well is to query for all the orders and opinions per judge within a specific time frame. For longer time frames, the query needs more time to process and give back the results.